lib/jekyll/feedmetatag.rb

```diff
@@ -0,0 +1,32 @@
+module Jekyll
+  class FeedMetaTag < Liquid::Tag
```

Comment:
I've always seen [files named with snake_case](https://github.com/bbatsov/ruby-style-guide#snake-case-files), meaning the file should be named `feed_meta_tag`.